### PR TITLE
Fix grammar in docs

### PR DESCRIPTION
**Pull request by DoubleOTheven**

### CODE REVIEW

File: docs/source/guide.rst [hunk 1]
```diff
@@ -24,7 +24,7 @@ See example configuration below:
 Browser wallet
 ==============
 
-To connect to a wallet in a web3-enabled browser, use ``activateBrowserWallet`` returned by ``useEthers()``. 
+To connect to a wallet in a web3-enabled browser, use ``activateBrowserWallet`` returned by ``useEthers()``.
```
Comment (DoubleOTheven) [inline docs/source/guide.rst:27]: My IDE fixes bad white space. No other changes...

File: docs/source/guide.rst [hunk 2]
```diff
@@ -242,7 +242,7 @@ To check if the hook reads data correctly, we need to prepare it first. We appro
 To test the hook we need to render it using ``renderWeb3Hook``. It works like ``renderHook`` from the `react-testing-library <https://testing-library.com/docs/react-testing-library/intro/>`_,
 but it wraps the hook into additional providers.
 
-React components update asynchronically. Reading data from the blockchain is also an async operation.
+React components are asynchronous. Reading data from the blockchain is also an async operation.
```
Comment (DoubleOTheven) [inline docs/source/guide.rst:245]: typo fix

File: docs/source/guide.rst [hunk 3]
```diff
@@ -304,23 +304,23 @@ is to use :ref:`useSendTransaction` hook, which returns a ``sendTransaction`` fu
...
+.. code-block:: javascript
 
   const handleClick = () => {
     setDisabled(true)
     sendTransaction({ to: address, value: utils.parseEther(amount) })
   }
 
 
-After that you can use state to check the state of your transtaction. State is of type :ref:`TransactionStatus`.
+After that you can use state to check the state of your transaction. State is of type :ref:`TransactionStatus`.
```
Comment (DoubleOTheven) [inline docs/source/guide.rst:323]: typo fix

File: docs/source/guide.rst [hunk 4]
```diff
@@ -511,5 +511,5 @@ As such the error can be handled in 3 ways:
   }
 
 
-Becouse useDApp defaults to read only connector ``error`` from ``useEthers()`` is only shown for few frames as such if you want to 
+Because useDApp defaults to read only connector ``error`` from ``useEthers()`` is only shown for few frames as such if you want to
```
Comment (DoubleOTheven) [inline docs/source/guide.rst:514]: typo fix

File: docs/source/guide.rst [hunk 5]
```diff
@@ -464,10 +464,10 @@ You have to remember that object in ``notifications`` array may not contain tran
  (that's why there is if statement).
 
 
-Handling wallet activation errrors
+Handling wallet activation errors
```
Comment (DoubleOTheven) [inline docs/source/guide.rst:467]: typo fix

**Review by jodator — APPROVED** [review]
Thanks!

**Pull request merged**